app/latigo/time_series_api/sensor_data_provider.py

```diff
@@ -134,11 +134,7 @@ def get_data_for_range(
             ts, err = self._fetch_data_for_id(id, time_range)
             # logger.info(f" D '{ts}, {err}'")
             if err or not ts:
-                if ts:
-                    msg = ts.get("latigo-ok", "Unknown failure")
-                else:
-                    msg = "No ts"
-                return None, err or msg
+                return None, err or "No ts"
```

Comment:
```suggestion
                return None, (err or "No ts")
```
Just for better readability.